JitsiTrackError.js

```diff
@@ -99,7 +102,8 @@ function JitsiTrackError(error, options, devices) {
                         || constraintName === 'minHeight'
                         || constraintName === 'maxHeight'
                         || constraintName === 'width'
-                        || constraintName === 'height')) {
+                        || constraintName === 'height'
+                        || constraintName === 'deviceId')) {
```

Comment:
will the error here actually have this as the constraintName?  won't it still be an error on width or height or something?

Reply:
No, this is the case where you unplug the camera. And there is actually no such deviceId. 
I was explaining you offline, that in such case there is no way to detect whether this failure is because of audio or video.
This will result if anybody now unplugs a camera which is set as default in settings and reloads we will always retry all resolutions before fallback to another device if any.